internal/ssm_test.go

```diff
@@ -31,13 +34,24 @@ func TestGetSSMParameter(t *testing.T) {
 	// Setup Test
 	mockSvc := &mockSSMClient{}
 
-	for k, v := range ssmTestValues {
-		t.Logf("Key: %s should have value: %s", k, v.expectedValue)
-		value, err := getSSMParameter(mockSvc, v.path, v.required, v.decrypt)
-		if err != nil {
-			t.Errorf("Expected %s , got %s", v.expectedValue, err)
-		} else if value != v.expectedValue {
-			t.Errorf("Expected %s , got %s", v.expectedValue, value)
+	for k, v := range fakeSSMStore {
+		if v.expectedValue != nil {
+			t.Logf("Key: %s should have value: %s", k, *v.expectedValue)
+		} else {
+			t.Logf("Key: %s should have value: nil", k)
+		}
+		value, err := getSSMParameter(mockSvc, k, v.defaultValue, false)
+
+		if v.expectedValue != nil && value != nil && *value == *v.expectedValue {
+			// Success
```

Comment:
Instead of just commenting success, please write why it passes.
e.g: // When the parameter does not exist an error is returned